modules/outconBidAdapter.js

```diff
@@ -0,0 +1,40 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {config} from '../src/config';
+const BIDDER_CODE = 'outcon';
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.pod || (bid.params.internalId && bid.params.publisher));
+  },
+  buildRequests: function(validBidRequests) {
+    for (let i = 0; i < validBidRequests.length; i++) {
+      let par = '';
+      if (validBidRequests[i].params.pod != undefined) par = 'get?pod=' + validBidRequests[i].params.pod;
+      else par = 'get?internalId=' + validBidRequests[i].params.internalId + '&publisher=' + validBidRequests[i].params.publisher;
+      if (validBidRequests[i].params.demo == true) par = par + '&demo=true';
+      return {
+        method: 'GET',
+        url: 'http://test.outcondigital.com:8048/ad/' + par,
```

Comment:
Is this URL just temporary or the permanent location?  If there are different domains that should be used if someone is doing a demo (or test) vs someone calling the production endpoint, you could control that logic through the adapter code and the bid params.  Other adapters have done something similar.

Reply:
Following your comment we decided to enable the other URLs, and we changed the parameters to specify the environment (env=test, env=api, env=stg).
The URLs are:
test.outcondigital.com
api.outcondigital.com
stg.outcondigital.com 

I tested the URL you said above and it works fine, maybe it was at a time of server maintenance, I get the following response:
{"impressionId":"5d812767bb00be0f847668df","ad":"5d42ea05306ea4689c2c8118","status":"ok","cpm":0.5,"cur":"USD","exp":300,"creatives":[{"url":"http://test.outcondigital.com/uploads/5d42ea05306ea4689c2c8112/outcon3.jpg","size":1,"width":1280,"height":720,"codec":"image/jpg"}],"type":"banner","duration":20,"campaign":"5d7fe03c306ea4018e095db2","trackingURL":"http://test.outcondigital.com:8048/ad/track?track=5d812767bb00be0f847668df"}

Reply:
I tried to make the request again and it succeeded this time (when I was off my company network).  However I encountered the following CORS error which prevented the request from being accepted (even though the response payload was there this time).

Can you please take a look?  I copied the error below for reference:
```
Access to XMLHttpRequest at 'http://test.outcondigital.com:8048/ad/get?internalId=12345678&publisher=5d5d66f2306ea4114a37c7c2&demo=true?' from origin 'http://test.localhost:9999' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.```

Reply:
Thanks for the feedback. I already changed the headers on the server to accept CORS policies, it should already be working

Reply:
I retested the adapter again, but I'm still seeing the CORS error (same as above).  Any ideas or recommedations?

<img width="1679" alt="Screen Shot 2019-09-23 at 12 03 11 PM" src="https://user-images.githubusercontent.com/31102355/65442519-47f03d80-ddfa-11e9-98d5-5487445ddb9f.png">

Reply:
Please check now again